main.yml

```diff
@@ -109,6 +125,16 @@ Resources:
           Value: "1"
       TemplateURL: ./s3bucket.yml
 
+  WorkerSG:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Parameters:
+        VPC: !If [CreateVPC, !GetAtt Network.Outputs.VPC, !Ref VpcId]
+      Tags:
+        - Key: valohai
+          Value: "1"
+      TemplateURL: ./worker-security-group.yml
```

Comment:
I'd keep it `worker-security-group.yml` because there is another SG that gets created inside the `worker-queue.yml` or then we'd have to move those around too 🤔 later?

Reply:
Ah it's only for workers. Sure then 👍